ls1intum/Artemis
# Feature/improved python templates

### Checklist

### Motivation and Context


According to [ARTEMIS-359](https://jirabruegge.in.tum.de/browse/ARTEMIS-359?filter=-1) there was no proper Python template included for Python programming exercises. At least, was not as extensive as the Java template.

### Description

I added a new Python template, including structural and behavioral tests. I used the same instructions as in the Java template in order to be consistent. In the browser, while creating a new exercise, the markdown and preview also get updated when choosing a different programming language.

### Steps for Testing


1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Create a new programming exercise for any course
4. Switch between languages, the template (incl. the preview) should change accordingy
